### ISSUE TITLE
Django Admin - Visual consistency fixes

### ISSUE THREAD

**Issue by reitermb** [raft-tech/TANF-app#1415]
**Description:**
**As a django admin user, I want elements to appear consistently cross-device and cross-browser**

On Chrome, there is no visible border for the search box or the action drop-down (see screenshot) which may make it difficult for users to perceive that these controls are actionable (or that they even exist). Suggest using the standard border design similar to what is in place in the filter dropdowns in right column of the page.

e.g. Search input field appears without its border on some devices (see screen capture below--Note lack of a border around search field):
![image](https://user-images.githubusercontent.com/28515874/139926479-6f77b58c-ba80-49ac-9ec1-31eb91575f0d.png) 

image

**Acceptance Criteria:** 
- [ ] All instances of visual differences across devices due to specific style values not being rendered are corrected
- [ ] Testing Checklist has been run and all tests pass
- [ ] Padding is fixed across the app to the pre-sidebar style
- [ ] Fix styles of `Add` buttons on list views
- [ ] Address warnings in console
- [ ] Fix size of filter dropdowns
- [ ] README is updated, if necessary

**Stretch Goal**
- [x] Hide the Add User button in Django Admin for TDRS

**Tasks:**
_Create a list of granular, specific work items that must be completed to deliver the desired outcomes of this issue_ 

- [ ] Force styles of input elements using css
- [ ] Run Testing Checklist and confirm all tests pass

**Supporting Documentation:**
[HackMD notes](https://hackmd.io/3zR7vj7GQ7uidJmLR-eCYw?view#Follow-On-Stuff)

**Comment by ADPennington** [comment]
11/17 Notes:
- for reproducing the problem, try unsetting any default settings in browser.

**Comment by abottoms-coder** [comment]
@reitermb Can you update the description of this ticket with the os/browser that exhibits the issues compared to Chrome? Should we be testing from Safari for example?
